### PR TITLE
Collection view wiring

### PR DESCRIPTION
**Pull request by denced**
https://trello.com/c/zzLH2jH4 Trello Card

Location:
-Map.storyboard
-DetailViewController.swift
-FavoritesView.storyboard
-FavoritesCollectionViewController.swift

Details:

Created connections to detail view from collections view
Displayed favorite button as selected on detail View is a favorite location is viewed.

Committed Features:

1 | Favorites button enabled when favorite location is viewed
2 | detail view displaying modally when favorite location is selected from collection view

### CODE REVIEW

**Comment by denced** [comment]
[Collection View Wiring](https://trello.com/c/zzLH2jH4/54-collection-view-wiring)

**Review by bronsonmullens — APPROVED** [review]
Good progress with the collection view. I'll look into diffable data as well.

File: labs-ios-starter/Map/DetailViewController.swift [hunk 1]
```diff
@@ -88,8 +73,19 @@ class DetailViewController: UIViewController {
...
+        if let cityController = cityController,
+           let city = city {
+            if cityController.favoriteCities.contains(city) {
+                print("we have a city: \(city)")
+                button0.isSelected = true
+            } else {
+                button0.isSelected = false
+            }
+        }
+
```
Comment (alext8900) [inline labs-ios-starter/Map/DetailViewController.swift:88]: So I was on the write track just had the code in the wrong place. Hmmm good job

**Review by alext8900 — APPROVED** [review]
Code looks good, you did a pretty good job.

**Review by alext8900 — APPROVED** [review]

**Pull request merged**